src/webui/src/static/model/trial.ts

```diff
@@ -110,7 +111,15 @@ class Trial implements TableObj {
         const endTime = this.info.endTime || new Date().getTime();
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         const duration = (endTime - this.info.startTime!) / 1000;
-
+        let accuracy;
+        if(this.acc !== undefined && this.acc.default !== undefined){
+            if(typeof this.acc.default === 'number'){
+                accuracy = JSON5.parse(this.acc.default);
```

Comment:
What's this for?

Reply:
to fix this state: when user give final result as  {tensor: xxx, data: xxx}

Reply:
Why to parse a number?